testsuite/features/support/constants.rb

```diff
@@ -1299,6 +1317,11 @@
         sle-micro-5.5-updates-x86_64
         sle-micro-5.5-devel-uyuni-client-x86_64
       ],
+    'sl-micro6.0' =>
+      %w[
+        sl-micro-6.0-pool-x86_64
+        sl-micro-6.0-devel-uyuni-client-x86_64
```

Comment:
These 2 are missing in the `TIMEOUT_BY_CHANNEL_NAME` list below.

Reply:
Added, thanks